tests/iscsi/iscsi_client.pm

```diff
@@ -116,7 +116,7 @@ sub run {
     record_info 'Systemd', 'Verify status of iscsi services and sockets';
     systemctl("is-active iscsid.service");
     systemctl("is-active iscsid.socket");
-    if (!is_sle('=12-SP4')) {
+    if (!is_sle('=12-SP4') && !is_sle('=12-SP5')) {
```

Comment:
I'm not sure about this change, how has it worked before and why did stop working? I guess we should understand the check better.

Reply:
I've checked manually and thing is that somehow this service is disabled in the image we use. If I perform fresh installation - service is active. In the test service is disabled (with vendor default - enabled). I don't know what disables it, as we don't do anything like that in the image creation job. So let's merge this one, as we are almost done with SLE 12 SP5, doesn't make much sense to invest time in fixing root cause.